src/applications/disability-benefits/all-claims/tests/schema/schema.unit.spec.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { Validator } from 'jsonschema';
+
+import formConfig from '../../config/form';
+import fullSchema from '../../config/schema';
+
+describe('526 all claims schema tests', () => {
+  const v = new Validator();
+  const files = fs.readdirSync(__dirname);
+  files.filter(file => file.endsWith('json')).forEach(file => {
+    it(`should validate ${file}`, () => {
+      const contents = JSON.parse(
+        fs.readFileSync(path.join(__dirname, file), 'utf8'),
+      );
+      const submitData = JSON.parse(
+        formConfig.transformForSubmit(formConfig, contents),
+      ).form526;
+      const result = v.validate(JSON.parse(submitData), fullSchema);
```

Comment:
I don't think this will catch the dynamic schema bits relating to object with boolean disability keys -> array of disability key strings.

Reply:
Ah, that's true. I can add a check to make sure all the dynamically generated properties (array of strings by this point) match condition names. Good call. :+1: